src/main/java/hudson/plugins/git/GitChangeSet.java

```diff
@@ -8,6 +8,7 @@
 import hudson.scm.ChangeLogSet;
 import hudson.scm.ChangeLogSet.AffectedFile;
 import hudson.scm.EditType;
+import org.apache.commons.lang.StringUtils;
```

Comment:
Please remove this import if it is unused.  It is not mentioned in the diff output, so I assume it is unused.